catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -7,32 +7,40 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
 
 We will keep this page up to date with the latest information as we make more progress.
 
-## What’s new? (updated June 2019)
+## What’s new? (updated 16 August 2019)
 
-We made it easier to get an overview of a book's content and review individual pages in detail. We are currently working to improve the relevance of your search results which will be released shortly.
+### More of the collection available to search
+We've made more of our digital collections available under an Audio/Video filter. You can now find: 
+- 4337 multi-volume works
+- 900 digitised videos and 
+- 3 audio files [(including an 1890 recording of Florence Nightingale)](https://wellcomecollection.org/works/tp9njewm)
```

Comment:
✨

Reply:
Does this imply that the multi-volume works are available under the Audio/Video filter?

Reply:
Ach good catch.

Reply:
fixed now.